matlab/schemas/+stimulus/Varma.m

```diff
@@ -59,23 +59,26 @@ function test()
             rng(cond.noise_seed)
```

Comment:
the `rng` should be used inside the `make` method so that it will work when called by the stimulus framework.